packages/data-service/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "data-service",
+  "module": "./src/index.js",
+  "version": "0.0.1",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "graphql": "0.13.2",
```

Comment:
@nLight after moving to the structure you asked, I ran into a problem I don't know how to solve. Those dependencies do not get installed when I do npm install on the original repository. I tried peerDependencies briefly was not able to solve it.

If you could please take a look that would be awesome.